apps/desktop-wallet/src/features/addressFiltering/addressFilteringHooks.tsx

```diff
@@ -16,77 +16,67 @@ You should have received a copy of the GNU Lesser General Public License
 along with the library. If not, see <http://www.gnu.org/licenses/>.
 */
 
-import { AddressHash } from '@alephium/shared'
-import { useEffect, useMemo, useState } from 'react'
+import { useMemo } from 'react'
 
-import { useAddressesUnlistedFTs } from '@/api/addressesUnlistedTokensHooks'
-import useAddressesTokensBalances from '@/api/apiDataHooks/useAddressesTokensBalances'
-import useAlphBalancesByAddress from '@/api/apiDataHooks/useAlphBalancesByAddress'
-import useFTList from '@/api/apiDataHooks/useFTList'
+import useFetchWalletBalancesAlphByAddress from '@/api/apiDataHooks/wallet/useFetchWalletBalancesAlphByAddress'
+import { useFetchWalletTokensBalancesByAddress } from '@/api/apiDataHooks/wallet/useFetchWalletBalancesTokensBy'
+import useFetchWalletFts from '@/api/apiDataHooks/wallet/useFetchWalletFts'
 import { useAppSelector } from '@/hooks/redux'
 import { selectAllAddresses, selectAllAddressHashes } from '@/storage/addresses/addressesSelectors'
 
 export const useFilterAddressesByText = (text = '') => {
   const allAddresses = useAppSelector(selectAllAddresses)
   const allAddressHashes = useAppSelector(selectAllAddressHashes)
-  const { data: fungibleTokenList } = useFTList()
-  const { data: unlistedFungibleTokens } = useAddressesUnlistedFTs()
-  const { data: addressesAlphBalances } = useAlphBalancesByAddress()
-  const { data: addressesTokensBalances } = useAddressesTokensBalances()
```

Comment:
Getting rid of the "old new" hooks in favor of the "new new" `useFetch...` ones.